packages/evm/src/evm.ts

```diff
@@ -632,6 +642,7 @@ export class EVM implements EVMInterface {
           }
```

Comment:
Should we also clearCaches when the code size exceeds maximum code size?

Reply:
we haven't written any code chunks here, so no need